Clarify "otherwise" in PR template.

tomalec opened the pull request:
### Changes proposed in this Pull Request:

- Clarify "otherwise" in PR template.

	Personally, I was not sure what "otherwise" in the sentence "Enter something here…, or leave it empty…, otherwise." means.
	Reading the `woorlease` code, I get that it means "remove the section".
- Describe how to get no changelog entry in PR template.
	Somewhat workaround 322-gh-woocommerce/woorelease
	//cc @ecgan, @nima-karimi 
Helps to avoid changelog entries like:
```
* Fix - <!--.
* Fix - Each line should start with `(Fix|Add|Tweak|Update) - `, for example:.
* Fix - Enter a summary of all changes in this Pull Request, which will be added to the changelog if accepted. .
* Fix - Optional.
```

### Detailed test instructions:

1. Create new PR, to see the template
2. Make a release :feelsgood: to see how the changelog is parsed.


### Changelog entry
<!-- test 323-gh-woocommerce/woorelease -->
### <!--Test 323-gh-woocommerce/woorelease-->

tomalec committed:
Clarify "otherwise" in PR template.

tomalec committed:
Clarify how to gen no changelog entry from PR,

Update the PR template, to emphasize that to workaround 322-gh-woocommerce/woorelease,
if we don't need any changelog entry, the comment itself needs to be removed as well.

Otherwise, it produces artifacts during the release.

tomalec committed:
Trim whitespaces in PR template

eason9487 commented:
I was thinking we could move the entire comments to the above of the header:
```
<!--
Optional.
Enter a summary of all changes [...omitted]
-->

### Changelog entry

> 
```

Thus, it could avoid these comments being parsed out from the **woorelease** if accidentally forgot to remove them.

tomalec commented:
Good idea.
It may be not 100% clear that the comment does not apply to the section it's in (as other comments), but the next one, but for sure that approach would be more bulletproof for the `woorelease` issue.

tomalec committed:
Move chanelog entry comment above the header,

to avoid 322-gh-woocommerce/woorelease,
as suggested at https://github.com/woocommerce/google-listings-and-ads/pull/961#issuecomment-899987027.

eason9487 reviewed (approved):
LGTM.
